content/src/Environment.ts

```diff
@@ -31,11 +31,10 @@ import { ValidationsFactory } from "./service/validations/ValidationsFactory";
 export const CURRENT_CONTENT_VERSION: EntityVersion = EntityVersion.V3
 const DEFAULT_STORAGE_ROOT_FOLDER = "storage"
 const DEFAULT_SERVER_PORT = 6969
-const DEFAULT_DCL_API_URL = "https://api.decentraland.zone/v1"
 export const DEFAULT_ETH_NETWORK = "ropsten"
+export const DEFAULT_DCL_PARCEL_ACCESS_URL = 'https://api.thegraph.com/subgraphs/name/nicosantangelo/watchtower'
```

Comment:
A few things about this change:
1. We need to update the .env on all servers. Ours (prod & dev) and the ones outside dcl
2. Is there a url that listens to ropsten? If so, should we use that as the default (as we did previously)?